astropy/io/votable/tests/test_coosys.py

```diff
@@ -55,25 +57,19 @@ def test_accept_refposition():
 
 
 def test_coosys_system():
-    """
-    Check that prior to v1.5, an error is issues for a system not in the hard-coded list.
-    With 1.5 the vocabulary is external and we're not checking that yet.
-    """
     for version, system, expected_warning in [
-        ("1.4", "ICRS", None),
-        ("1.4", "supergalactic", None),
```

Comment:
shouldn't we still keep these?

Reply:
I'll add them back, but it is supposed to be the same in version 1.4 and 1.5. The test felt strange to me